src/FirstSteps/RPG/Adventures/Hell.cs

```diff
@@ -0,0 +1,63 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Tools;
+using Spectre.Console;
+using System;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public class Hell
+    {
+        public void FightWithBoss(Hero hero)
+        {
+            var diablo = Boss.CreateDiablo();
+
+            Display.DefaultText($"You entered to the Hell!");
+
+            do
+            {
+                string userInput = AnsiConsole.Prompt(
+                     new SelectionPrompt<string>()
+                       .Title($" \nChose what u want to do! {Emoji.Known.CrossedSwords}")
+                       .AddChoices("Boss stats", "Hit the boss!", "Run"));
+
+                switch (userInput)
+                {
+                    case "Hit the boss!":
+                        var number = new Random().Next(1, 5);
+                        Console.WriteLine($"You rolled {number}");
+                        if (number >= 2)
+                        {
+                            diablo.TakeDamage(hero.DealDamage());
```

Comment:
You missed one previous comment. `You always do the DealDamage calculation twice. What if it was an expensive operation took 5s?`.